### PR TITLE
fix(image): properly lock loremflickr.com images

### PR DESCRIPTION
**Pull request by wesen**
https://loremflickr.com requires the lock= parameter to be given
to serve an identical image (at least as long as its cache is not
updated).

### CODE REVIEW

**Review by ST-DDT — APPROVED** [review]

File: src/modules/image/index.ts [hunk 1]
```diff
@@ -111,7 +111,9 @@ export class ImageModule {
     }
 
     if (randomize) {
-      url += `?${this.faker.datatype.number()}`;
+      url += `?lock=${this.faker.datatype.number()}`;
+    } else {
+      url += `?lock=0`;
```
Comment (ST-DDT) [inline src/modules/image/index.ts:116]: This changes the current behavior. Currently, returning a random picture is intentional.
In the next major version we plan to change the default of randomize to true.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: src/modules/image/index.ts (hunk 1 above)
Comment (wesen) [reply src/modules/image/index.ts:116]: does this mean that randomize = false still randomizes the image? I'm sorry if I misunderstood the meaning.

Comment (ST-DDT) [reply src/modules/image/index.ts]: Yes, `randomize === false` => no `lock` url suffix.

File: src/modules/image/index.ts [hunk 2]
```diff
@@ -39,9 +39,9 @@ export class ImageModule {
    * @param randomize Whether to randomize the image or not. Defaults to `false`.
    *
    * @example
-   * faker.image.image() // 'https://loremflickr.com/640/480/city'
-   * faker.image.image(1234, 2345) // 'https://loremflickr.com/1234/2345/sports'
-   * faker.image.image(1234, 2345, true) // 'https://loremflickr.com/1234/2345/nature?56789'
+   * faker.image.image() // 'https://loremflickr.com/640/480/city?lock=0'
+   * faker.image.image(1234, 2345) // 'https://loremflickr.com/1234/2345/sports?lock=0'
```
Comment (ST-DDT) [inline src/modules/image/index.ts:43]: Please adjust the examples accordingly.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: src/modules/image/index.ts (hunk 1 above)
Comment (wesen) [reply src/modules/image/index.ts]: Updated!

**Review by ST-DDT — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Pull request merged**